task11.txt

```diff
@@ -0,0 +1,109 @@
+You are now connected to database "nc_news_test" as user "paulosghirmai".
+ slug  |          description           
+-------+--------------------------------
+ mitch | The man, the Mitch, the legend
+ cats  | Not dogs
+ paper | what books are made of
+(3 rows)
+
+   username    |    name    |                                avatar_url                                 
+---------------+------------+---------------------------------------------------------------------------
```

Comment:
try not to push files not needed for your app!

either avoid `git add`-ing them and/or add them to your `.gitignore`